packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -198,6 +201,9 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       body.device.countryCode = this.webex.meetings.geoHintInfo.countryCode;
       // @ts-ignore
       body.device.regionCode = this.webex.meetings.geoHintInfo.regionCode;
+      body.device.localIp =
+        // @ts-ignore
+        anonymizeIPAddress(this.webex.meetings.geoHintInfo?.clientAddress) || undefined;
```

Comment:
nitpick: Why the optional chaining operator? If `getHintInfo` was undefined at this point, it shouldn't be in this block due to the guard statement, and even if that wasn't there, the previous lines would have error-ed before this line executed.

Reply:
yeah I thought about that too, im happy to remove this.

Reply:
removed